blog/Welcome.md

```diff
@@ -72,7 +72,7 @@ Never made an open source contribution before? Wondering how contributions work
 
 ### Where can I go for help 🤨?
 
-If you need help, you can ask questions on our [discord](https://dicord.com) channel.
+If you need help, you can ask questions on our [discord](https://discord.com) channel.
```

Comment:
We have to add whatsapp link to it.

Reply:
Instead of Discord, Add Whatsapp [link](https://chat.whatsapp.com/CQ0LXqibJjCESfdnWrIbRR) to it.

Reply:
okay!